src/components/UploadSection.svelte

```diff
@@ -14,12 +21,54 @@
         }
     });
 
+    async function sendFiles() {
+        let postRequests: Array<PostInsertRequestRequest> = [];
+        let postResponse: PostInsertRequestResponse[];
+
+        for (let i: number = 0; i < files.length; i++) {
+            let exifData = null;
+
+            try {
+                let exifData = await exifr.parse(files[i]);
+            } catch (e) {
+                console.error(e);
+            }
+
+            let takenAtSec = null;
+
+            if (exifData) {
+                takenAtSec = exifData.DateTimeOriginal.valueOf();
+            }
+
+            let postRequest: PostInsertRequestRequest = {
+                ObjectKey: files[i].name,
+                SizeBytes: files[i].size,
+                TakenAtSec: takenAtSec,
+            };
+
+            postRequests.push(postRequest);
+        }
+        try {
+            postResponse = await postInsertRequest(postRequests);
+            const loopTest = [];
+            for (let i = 0; i < files.length; i++) {
+                loopTest.push(
+                    pushMediaFilesToS3(postResponse[i].UploadUrl, files[i])
+                );
+            }
+            await Promise.all(loopTest);
```

Comment:
Ok for now, but we should remember to add error handling later

Reply:
Does this try catch not do the issue ? or do more than just logging error?